src/main/resources/com/google/api/codegen/java/LICENSE

```diff
@@ -0,0 +1,201 @@
+                            Apache License
```

Comment:
This text is already available at com.google.api.codegen/LICENSE.snip - could we make it so there is only one copy?